internal/gatewayapi/route.go

```diff
@@ -1226,16 +1226,68 @@ func (t *Translator) processDestination(backendRefContext BackendRefContext,
 			"Mixed endpointslice address type for the same backendRef is not supported")
 	}
 
+	destinationFilters := t.processDestinationFilters(routeType, backendRefContext, parentRef, route, resources)
 	ds = &ir.DestinationSetting{
 		Weight:      &weight,
 		Protocol:    protocol,
 		Endpoints:   endpoints,
 		AddressType: addrType,
 		TLS:         backendTLS,
+		Filters:     destinationFilters,
```

Comment:
can we add a nil check here just to avoid all the `filters: {}` changes in this PR?

Reply:
+1

Reply:
sorry, problem was withing getBackendFilters func